session4/src/itschool/homework/ex1/BinaryArithmeticOperators.java

```diff
@@ -0,0 +1,26 @@
+package itschool.homework.ex1;
+
+import java.util.Scanner;
+
+public class BinaryArithmeticOperators {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int num1, num2, res;
+
+        System.out.println("Enter the First Number: ");
+        num1 = scan.nextInt();
+        System.out.println("Enter the Second Number: ");
+        num2 = scan.nextInt();
+
+        res = num1 + num2;
+        System.out.println("Addition Result = " + res);
+        res = num1 - num2;
+        System.out.println("Subtraction Result = " + res);
+        res = num1 * num2;
+        System.out.println("Multiplication Result = " + res);
+        res = num1 / num2;
+        System.out.println("Division Result = " + res);
+        res = num1 % num2;
```

Comment:
so nice to see you are reusing the same `res` variable! Well done! You made some efficient work here saving memory by not creating extra variables that go out of scope immediately.  
However, keep in mind (it is not the case here) that this approach has also disadvantages - one of them would be that you might need this `res` variable as result of an addition, let's say, somewhere in this program, and then you would have to refactor, meaning you will need a dedicate variable for addition only.